File: app/src/main/java/com/example/theloop/network/FunFactApiService.java
package com.example.theloop.network;

import com.example.theloop.models.FunFactResponse;
import retrofit2.Call;
import retrofit2.http.GET;

public interface FunFactApiService {
    @GET("random.json")
    Call<FunFactResponse> getRandomFact(@retrofit2.http.Query("language") String language);
}

File: app/src/main/java/com/example/theloop/network/FunFactApiService.kt
package com.example.theloop.network

import com.example.theloop.models.FunFactResponse
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Query

interface FunFactApiService {
    @GET("random.json")
    suspend fun getRandomFact(@Query("language") language: String): Response<FunFactResponse>
}
